tests/integration/test_passwords.py

```diff
@@ -60,6 +62,123 @@ def test_forgot_password_post_success(self, mock_request, mock_notify):
         mock_notify.assert_called_once()
         self.assertTrue("Check your email".encode() in response.data)
 
+    @requests_mock.mock()
+    @patch("frontstage.controllers.notify_controller.NotifyGateway.request_to_notify")
+    def test_forgot_password_post_success_with_4_try_warning(self, mock_request, mock_notify):
```

Comment:
Not sure about the name of this test, maybe something reffering to 1 try remaining rather than 4 try, it's not immediately obvious what that is referring to